index.html

```diff
@@ -2,11 +2,134 @@
   <head>
     <meta charset="utf-8">
+    <meta name="keywords" content="Седона, туризм, отели">
+    <meta name="description" content="описание города Седона">
+    <link rel="stylesheet" href="">
     <title>HTML Academy: Седона</title>
   </head>
+
   <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
+     <header>
+      <nav>
+          <ul>
+              <li><a href="">Информация</a></li>
+              <li><a href="">Фото и видео</a></li>
+              <li><a href="">Карта штата</a></li>
+              <li><a href="">Гостиницы</a></li>
+          </ul>
+          <img src="https://via.placeholder.com/138x70" alt="логотип"> 
+      </nav>
+     </header>
+
+     <main>
+      <section>
+       <p>Welcome to the Gorgeous</p>
+       <h1>Sedona</h1>
+       <em>"Because the Grand Canyon sucks!"</em>
+      </section>
+      <section>
+       <h2>Седона-небольшой городок в Аризоне,<br>
+          заслуживающий большего!</h2> 
+       <p>Рассмотрим 5 причин, по которым Седона круче, чем Гранд Каньон!</p>
+      </section>
+      <section>
+       <ul>
+        <li>
+          <h3>Настоящий городок</h3>
+          <span>№1</span>
+          <p>Седона - не аттракцион для туристов, там<br>
+          течёт своя жизнь</p>
+          <img src="https://via.placeholder.com/800x256" alt="Фото Седоны">
+          <ul>
+            <li>
+              <h4>Жильё</h4>
+              <p>Рекомендуем пожить в настоящем<br>
+               мотеле, всё как в кино!</p>
+            </li>
+            <li>
+              <h4>Еда</h4>
+              <p>Всегда заказывайте фирменный бургер,<br>
+               вы не разочаруетесь!</p>
+            </li>
+            <li>
+              <h4>Сувениры</h4>
+              <p>Не только китайского, но и местного
+                производства!</p>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <h3>Там есть мост дьявола</h3>
+          <span>№2</span>
+          <p>Да, по нему можно пройти! Если конечно<br>
+              вы осмелитесь</p>
+          <img src="https://via.placeholder.com/800x256" alt="Окрестности Седоны">
+        </li>
+        <li>
+          <h3>Небольшая площадь</h3>
+          <span>№3</span>
+          <p>все достопримечательности<br>
+              находятся очень близко</p>
+        </li>
+        <li>
+          <h3>Красивая дорога</h3>
+          <span>№4</span>
+          <p>ехать в Седону из Лас-Вегаса совсем<br>
+              не скучно!</p>
+        </li>
+        <li>
+          <h3>Мало туристов</h3>
+          <span>№5</span>
+          <p>большинство едет в Гранд Каньон<br>
+              и толпится там</p>
+        </li>
+
+        </ul>
+      </section>
+      <section>
+       <h2>Заинтересовались?</h2>
+       <p>Укажите предполагаемые даты поездки,<br>
+            и мы покажем вам лучшие предложения гостиниц в Седоне  </p>
```

Comment:
✂️  лишние коды символов

Reply:
нужно убрать лишние символы, скорее всего это автодобавление при копировании из фигмы
```suggestion
          <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
```